cordova-lib/src/plugman/plugman.js

```diff
@@ -169,35 +160,6 @@ plugman.commands =  {
             }
         });
     },
-
-    'publish'  : function(cli_opts) {
```

Comment:
I wonder if these commands should still be exposed but with a meaningful console.error message on either having to use the older version of plugman < current or if we decide to make CPR read-only - stating that as such.